Code/Legacy/CrySystem/DebugCallStack.cpp

```diff
@@ -896,13 +778,40 @@ static bool IsFloatingPointException(EXCEPTION_POINTERS* pex)
     }
 }
 
-int DebugCallStack::PrintException(EXCEPTION_POINTERS* exception_pointer)
+DebugCallStack::UserPostExceptionChoice DebugCallStack::AskUserToRecoverOrCrash(EXCEPTION_POINTERS* exception_pointer)
 {
-    return (int)DialogBoxParam(gDLLHandle, MAKEINTRESOURCE(IDD_CRITICAL_ERROR), NULL, DebugCallStack::ExceptionDialogProc, (LPARAM)exception_pointer);
+    if (exception_pointer->ExceptionRecord->ExceptionFlags & EXCEPTION_NONCONTINUABLE)
+        return UserPostExceptionChoice::Exit;
```

Comment:
missing curly brace [sig-core style guideline](https://github.com/o3de/sig-core/blob/main/governance/Coding-Standards-and-Style-Guide.md#use-of-curly-braces) 
```suggestion
    if (exception_pointer->ExceptionRecord->ExceptionFlags & EXCEPTION_NONCONTINUABLE)
    {
        return UserPostExceptionChoice::Exit;
    }
```